packages/core/phase1/enrichAho/enrichFunctions/enrichWithPncQuery.ts

```diff
@@ -47,19 +47,18 @@ const clearPNCPopulatedElements = (aho: AnnotatedHearingOutcome): void => {
 }
 
 export default async (
-  annotatedHearingOutcome: AnnotatedHearingOutcome,
```

Comment:
I think we were trying to avoid acronyms

Reply:
I'll put this back